app/controllers/application.js

```diff
@@ -1,16 +1,22 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import { set } from "@ember/object";
+import { computed } from '@ember/object';
 import move from 'ember-animated/motions/move';
 import { easeInAndOut } from 'ember-animated/easings/cosine';
 
 export default Controller.extend({
   dj             : service(),
   hifi           : service(),
+  cookies        : service(),
   currentStream  : service(),
   links: [ { 'href': null, 'nav-slug': 'listen', 'title': 'Listen'}, { 'href': null, 'nav-slug': 'playlist-history', 'title': 'Playlist History'} ],
 
   showPlayer: true,
+  showOnboardMessage: computed('closed', function() {
+    let cookieService = this.get('cookies');
+    return !cookieService.exists('showOnboardMessage');
```

Comment:
you don't actually have to use `get` in the version of ember we're using, so you can change this to a one-liner: 

`return !this.cookies.exists('showOnboardMessage')`